src/dao/LOCsManagerDAO.js

```diff
@@ -34,7 +35,7 @@ class LOCsManagerDAO extends AbstractContractDAO {
         let settingIndex = Setting.get(settingName)
         promises.push(loc.getString(settingName, account).then(r => {
           if (r === value) return
-          return deployed.setLOCString(data.address, settingIndex, this._toBytes32(value), {from: account})
+          return deployed.setLOCString(data.address, settingIndex, bytes32(value), {from: account})
```

Comment:
Because AbstractContractDAO._toBytes32 some times returns wrong values, for example: "X|}"